Engines/Wine/Engine/Implementation/script.js

```diff
@@ -444,21 +447,25 @@ module.default = class WineEngine {
             ldPath = userData.ldPath + ldPath;
         }
 
+        let runtimePath = "";
+        let runtimePath64 = "";
+        if (this._useRuntime) {
+            runtimePath = this._wineEnginesDirectory + "/runtime/lib/";
+            runtimePath64 = this._wineEnginesDirectory + "/runtime/lib64/";
+        }
+
         if (architecture == "amd64") {
             ldPath =
-                this._wineEnginesDirectory +
-                "/runtime/lib64/:" +
-                this._wineEnginesDirectory +
-                "/runtime/lib/:" +
+                runtimePath64 + ":" +
+                runtimePath + ":" +
                 this.getLocalDirectory(subCategory, version) +
                 "/lib64/:" +
                 this.getLocalDirectory(subCategory, version) +
                 "/lib/:" +
```

Comment:
Can we move these two paths also to variables?

Reply:
We could but honestly I do not see the advantage of it. Also it's not related to this PR at all.

Reply:
It makes the composition of the final `ldPath` string easier to read

Reply:
Done.